jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -2165,7 +2168,7 @@ public AsyncContext startAsync() throws IllegalStateException
         HttpChannelState state = getHttpChannelState();
         if (_async == null)
             _async = new AsyncContextState(state);
-        AsyncContextEvent event = new AsyncContextEvent(_context, _async, state, this, this, getResponse());
+        AsyncContextEvent event = new AsyncContextEvent(_context, _async, state, this, this, getResponse(), false);
```

Comment:
I wonder if you shouldn't keep the old constructor here, and not use an extra boolean argument.